### ISSUE TITLE
forceUpdate is not consistently implemented

### ISSUE THREAD

**Issue by maddy2101** [NamelessCoder/fluid-documentation-generator#6]
Some files are recreated, some are not. This should please be consistent as to avoid a mixed state of old and new files in the output.

**Comment by NamelessCoder** [comment]
I agree this flag isn't completely obvious - but it is actually intended to be completely up to each individual exporter if it wants to respect the flag. In the implementations that ship with the package there is partial respect for it: overview pages will always be generated (since they likely include links to things like new package versions in the left menu) but the deeper individual pages for ViewHelpers will not unless you force them (so that if you had to make an update in an XSD you *can* regenerate also the files that normally don't ever need to be regenerated).

Hope that makes sense. I think the alternative to this rather naive strategy, would be something like storing checksums of various asset lists like packages and their versions and as default strategy only regenerate files if a checksum mismatches or file doesn't exist. But of course that takes a much more detailed and shared implementation than just leaving it up to each exporter...
